code/modules/food_and_drinks/food/snacks_frozen.dm

```diff
@@ -43,6 +43,143 @@
+	list_reagents = list(/datum/reagent/consumable/ice_cream/strawberry = 10)
+	filling_color = "#EFB8B8"
+	tastes = list("ice cream" = 1, "strawberries" = 1)
+	foodtype = DAIRY | SUGAR | FRUIT
+
+/obj/item/reagent_containers/food/snacks/ice_cream_scoop/blue
+	name = "blue ice cream scoop"
+	desc = "A faintly blue ice cream flavor; it is notorious for its ability to stain."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "icecream_blue"
+	bonus_reagents = list(/datum/reagent/consumable/ice_cream/blue = 2)
+	list_reagents = list(/datum/reagent/consumable/ice_cream/blue = 10)
+	filling_color = "#B8C5EF"
+	tastes = list("ice cream" = 1, "blue" = 1)
+	foodtype = DAIRY | SUGAR | ALCOHOL
+
+/obj/item/reagent_containers/food/snacks/ice_cream_scoop/lemon_sorbet
+	name = "lemon sorbet scoop"
+	desc = "An ancient frozen treat supposedly invented by the Persians that is still enjoyed today."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "icecream_lemon sorbet"
+	bonus_reagents = list(/datum/reagent/consumable/ice_cream/lemon_sorbet = 2)
+	list_reagents = list(/datum/reagent/consumable/ice_cream/lemon_sorbet = 10)
+	filling_color = "#D4DB86"
+	tastes = list("ice cream" = 1, "lemons" = 1)
+	foodtype = DAIRY | SUGAR | FRUIT
+
+/obj/item/reagent_containers/food/snacks/ice_cream_scoop/caramel
+	name = "caramel ice cream scoop"
+	desc = "Ice cream that has been flavored with caramel; a treat for sugar lovers."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "icecream_caramel"
+	bonus_reagents = list(/datum/reagent/consumable/ice_cream/caramel = 2)
+	list_reagents = list(/datum/reagent/consumable/ice_cream/caramel = 10)
+	filling_color = "#BC762F"
+	tastes = list("ice cream" = 1, "caramel" = 1)
+	foodtype = DAIRY | SUGAR
+
+/obj/item/reagent_containers/food/snacks/ice_cream_scoop/banana
+	name = "banana ice cream scoop"
+	desc = "The ice cream of choice for clowns everywhere. Honk!"
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "icecream_banana"
+	bonus_reagents = list(/datum/reagent/consumable/ice_cream/banana = 2)
+	list_reagents = list(/datum/reagent/consumable/ice_cream/banana = 10)
+	filling_color = "#DEDE00"
+	tastes = list("ice cream" = 1, "banana" = 1)
+	foodtype = DAIRY | SUGAR | FRUIT
+
+/obj/item/reagent_containers/food/snacks/ice_cream_scoop/orange_creamsicle
+	name = "orange creamsicle scoop"
+	desc = "An ice cream flavor made after a popular popsicle flavor. It is not quite the same off the stick..."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "icecream_orangesicle"
+	bonus_reagents = list(/datum/reagent/consumable/ice_cream/orange_creamsicle = 2)
+	list_reagents = list(/datum/reagent/consumable/ice_cream/orange_creamsicle = 10)
+	filling_color = "#D8B258"
+	tastes = list("ice cream" = 1, "oranges" = 1)
+	foodtype = DAIRY | SUGAR | FRUIT
+
+/obj/item/reagent_containers/food/snacks/ice_cream_scoop/peach
+	name = "peach ice cream scoop"
+	desc = "Ice cream flavored with peaches; it is rather uncommon due to wizards buying up most of it."
```

Comment:
i don't quite understand the joke on this one, care to elaborate?

Reply:
If I recall correctly, maint peaches have it in their description that wizards bought up all the peaches because they're magical or something like that.